src/utils.js

```diff
@@ -24,7 +24,12 @@ function some(pred, obj) {
  */
 var sortByFunc =
     (prop) =>
-        (a, b) => a[prop] < b[prop] ? -1 : a[prop] > b[prop] ? 1 : 0;
+        (a, b) => {
+          if (b[prop] === null && a[prop] === null) { return 0 }
+          if (a[prop] === null) { return -1 }
+          if (b[prop] === null) { return 1 }
+          return a[prop] < b[prop] ? -1 : a[prop] > b[prop] ? 1 : 0;
+        }
```

Comment:
Could you make it that it returns 1 or -1 only once? Similar to what underscorejs does: https://github.com/jashkenas/underscore/blob/master/underscore.js#L392